db/schema.rb

```diff
@@ -10,9 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2021_11_09_150717) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "monsters", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "habitat"
+    t.text "description", null: false
+    t.string "image", null: false
```

Comment:
Doesn't look like you have image uploading yet in your app - I would remove this for now. Most likely you will be working on a feature first for being able to add new Monsters, and here you're requiring an image when creating a Monster. Let's break that down into smaller features. Once you have a working form for Monsters, then you can work in the ability for uploading an image within that form. At that point, you can come back to your Monster table and add the image column